packages/data-api/scripts/patches/1_0_0/20211207200858-AddRenameMaterializedViewLog.js

```diff
@@ -0,0 +1,155 @@
+let dbm;
```

Comment:
I've written equivalents to this patch in `mvBESFunctions.sql` directly, but I don't currently have permissions to push to the `pg-mv-refresh` repo